docker-compose-bahmniemr.yml

```diff
@@ -99,7 +99,7 @@ services:
       - OPENMRS_DB_USER=${OPENMRS_DB_USER}
       - OPENMRS_DB_PASSWORD=${OPENMRS_DB_PASSWORD}
       - BAHMNI_TEST_ORDER_TYPE_UUID=${EIP_TEST_ORDER_TYPE_UUID}
-    image: mekomsolutions/eip-client:sha-b2f32ae
+    image: mekomsolutions/eip-client:2.3.0
```

Comment:
This is for bahmni, uses the old version of `eip-client` (doesn't work with the new one)